gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/alttext/AltTextSection.kt

```diff
@@ -0,0 +1,238 @@
+package com.gravatar.quickeditor.ui.alttext
+
+import androidx.activity.compose.BackHandler
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.material3.LocalTextStyle
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.compose.LocalLifecycleOwner
+import androidx.lifecycle.repeatOnLifecycle
+import coil.compose.AsyncImage
+import com.gravatar.quickeditor.R
+import com.gravatar.quickeditor.ui.avatarpicker.AltTextSectionUiState
+import com.gravatar.quickeditor.ui.avatarpicker.AvatarPickerAction
+import com.gravatar.quickeditor.ui.avatarpicker.AvatarPickerEvent
+import com.gravatar.quickeditor.ui.avatarpicker.AvatarPickerViewModel
+import com.gravatar.quickeditor.ui.avatarpicker.AvatarUpdateType
+import com.gravatar.quickeditor.ui.avatarpicker.errorStringRes
+import com.gravatar.quickeditor.ui.components.QEButton
+import com.gravatar.quickeditor.ui.components.QESectionTitle
+import com.gravatar.quickeditor.ui.extensions.QESnackbarHost
+import com.gravatar.quickeditor.ui.extensions.SnackbarType
+import com.gravatar.quickeditor.ui.extensions.showQESnackbar
+import com.gravatar.restapi.models.Avatar
+import com.gravatar.ui.GravatarTheme
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import java.net.URI
+import java.net.URL
+
+@Composable
+internal fun AltTextSection(
+    onBackPressed: () -> Unit,
+    viewModel: AvatarPickerViewModel,
+    modifier: Modifier = Modifier,
+) {
+    BackHandler {
+        onBackPressed()
+    }
```

Comment:
⚠️ This works well before the keyboard is opened. However, after that, the bottom sheet is dismissed instead of being handled here. If that's OK with you, I'll create a ticket for this.

Reply:
That's weird... we can ticket this.

Reply:
[Done](https://github.com/Automattic/Gravatar-SDK-Android/issues/513)